packages/core/src/domain/tracekit/computeStackTrace.ts

```diff
@@ -1,556 +1,128 @@
-  const winjs =
-    /^\s*at (?:((?:\[object object\])?.+) )?\(?((?:file|ms-appx|https?|webpack|blob):.*?):(\d+)(?::(\d+))?\)?\s*$/i
-  /* eslint-enable  max-len */
-
-  // Used to additionally parse URL/line/column from eval frames
-  let isEval
-  const geckoEval = /(\S+) line (\d+)(?: > eval line \d+)* > eval/i
-  const chromeEval = /\((\S*)(?::(\d+))(?::(\d+))\)/
-  const lines = stacktrace.split('\n')
-  const stack = []
-  let submatch
-  let parts
-  let element
-
-  for (let i = 0, j = lines.length; i < j; i += 1) {
-    if (chrome.exec(lines[i])) {
-      parts = chrome.exec(lines[i])!
-      const isNative = parts[2] && parts[2].indexOf('native') === 0 // start of line
-      isEval = parts[2] && parts[2].indexOf('eval') === 0 // start of line
-      submatch = chromeEval.exec(parts[2])
-      if (isEval && submatch) {
-        // throw out eval line/column and use top-most line/column number
-        parts[2] = submatch[1] // url
-        parts[3] = submatch[2] // line
-        parts[4] = submatch[3] // column
-      }
-      element = {
-        args: isNative ? [parts[2]] : [],
-        column: parts[4] ? +parts[4] : undefined,
-        func: parts[1] || UNKNOWN_FUNCTION,
-        line: parts[3] ? +parts[3] : undefined,
-        url: !isNative ? parts[2] : undefined,
-      }
-    } else if (winjs.exec(lines[i])) {
-      parts = winjs.exec(lines[i])!
-      element = {
-        args: [],
-        column: parts[4] ? +parts[4] : undefined,
-        func: parts[1] || UNKNOWN_FUNCTION,
-        line: +parts[3],
-        url: parts[2],
-      }
-    } else if (gecko.exec(lines[i])) {
-      parts = gecko.exec(lines[i])!
-      isEval = parts[3] && parts[3].indexOf(' > eval') > -1
-      submatch = geckoEval.exec(parts[3])
-      if (isEval && submatch) {
-        // throw out eval line/column and use top-most line number
-        parts[3] = submatch[1]
-        parts[4] = submatch[2]
-        parts[5] = undefined! // no column when eval
-      } else if (i === 0 && !parts[5] && !isUndefined((ex as any).columnNumber)) {
-        // FireFox uses this awesome columnNumber property for its top frame
-        // Also note, Firefox's column number is 0-based and everything else expects 1-based,
-        // so adding 1
-        // NOTE: this hack doesn't work if top-most frame is eval
-        stack[0].column = ((ex as any).columnNumber as number) + 1
-      }
-      element = {
-        args: parts[2] ? parts[2].split(',') : [],
-        column: parts[5] ? +parts[5] : undefined,
-        func: parts[1] || UNKNOWN_FUNCTION,
-        line: parts[4] ? +parts[4] : undefined,
-        url: parts[3],
-      }
-    } else {
-      continue
-    }
-
-    if (!element.func && element.line) {
-      element.func = UNKNOWN_FUNCTION
-    }
-    stack.push(element)
+export function computeStackTraceOfCaller(): StackTrace {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏it seems to only be used by the test, do we need to keep it?